app/pods/components/route-register/template.hbs

```diff
@@ -0,0 +1,53 @@
+<SectionHeader
+  @backIcon="home"
+  @backRoute="landing"
+  @icon="user-plus"
+>
+  Register
+</SectionHeader>
+
+<div local-class="register">
+  <div local-class="row">
+    <label>
+      Email
+      <Input
```

Comment:
You might as well use a normal HTML input if you're not relying on two way bindings. Isn't passing `this.email` as the value just going to be setting that twice?